packages/ux-capture/src/View.js

```diff
@@ -15,6 +15,9 @@ function View(props) {
 // TODO: determine if we need to support appending new marks
 // to exisiting zones or new zones or both
 View.prototype.update = function (zoneConfigs) {
+	// Append configuration, same way we append the zones themselves
+	this.props.zoneConfigs.push.apply(this.props.zoneConfigs, zoneConfigs);
```

Comment:
is there a reason to use this `this.props.zoneConfigs.push.apply(this.props.zoneConfigs, zoneConfigs)` syntax instead of just `this.props.zoneConfigs.push(...zoneConfigs)`?

Reply:
Now that I am thinking about it - it was probably because spread was a new feature which we didn't want to use when we were not relying on webpack to compile it.

Same goes for using prototypal inheritance, which should probably be rewritten in a more modern syntax for better readability.